entity-service-api/src/main/proto/org/hypertrace/entity/data/service/v1/entity_data_request.proto

```diff
@@ -54,3 +57,13 @@ message AttributeFilter {
   AttributeValue attributeValue = 3;
   repeated AttributeFilter childFilter = 4;
 }
+
+message OrderByExpression {
+  string name = 1;
+  SortOrder order = 2;
+}
+
+enum SortOrder {
+  ASC = 0;
```

Comment:
Do you want ASC to be the default?

Reply:
That's what I'm thinking. 

If it's unspecified we assumes ascending. Otherwise, the API won't be stringent enough. supposed I have: 
SortOrder {
   Unspecified = 0; 
   Asc = 1;
   Desc = 2;
}

If it's unspecified, then, in the implementation it would have to make it as Ascending Or throw a RuntimeException such as IllegalArgumentException. This is difficult to test for the user of this API.